### PR TITLE
add links to our feedback form

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

Makes our feedback link more prominent

### :camera_flash: Screenshots
Screenshot 2023-09-20 at 08 25 11
Screenshot 2023-09-20 at 08 25 00
Screenshot 2023-09-20 at 08 24 55

### :link: External links

Jira ticket: [HDS-2174](https://hashicorp.atlassian.net/browse/HDS-2174)

***

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-2174]: https://hashicorp.atlassian.net/browse/HDS-2174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: website/app/components/doc/page/footer.hbs [hunk 1]
```diff
@@ -21,6 +21,9 @@
       <li class="doc-page-footer__nav-item">
         <a href="https://www.hashicorp.com/privacy" target="_blank" rel="noopener noreferrer">Privacy</a>
       </li>
+      <li class="doc-page-footer__nav-item">
+        <a href="https://go.hashi.co/hds-feedback" target="_blank" rel="noopener noreferrer">Helios Feedback</a>
```
Comment (didoo) [inline website/app/components/doc/page/footer.hbs:25]: One says `Helios Feedback` the other `Share feedback`. Probably better to standadize them.
```suggestion
        <a href="https://go.hashi.co/hds-feedback" target="_blank" rel="noopener noreferrer">Share feedback</a>
```

File: website/docs/about/support.md [hunk 2]
```diff
@@ -28,4 +28,8 @@ Users external to HashiCorp can [file issues in GitHub](https://github.com/hashi
 
 ## Office hours
 
-Office hours are a great way to get feedback on your project or help to get unstuck in a code migration. [Sign up for an appointment](https://docs.google.com/document/d/1LdyQznltt4VCbGGGDuDkARRRKXI4laCHxHhY8dJy4wE/edit?usp=sharing) at the next office hours.
\ No newline at end of file
+Office hours are a great way to get feedback on your project or help to get unstuck in a code migration. [Sign up for an appointment](https://docs.google.com/document/d/1LdyQznltt4VCbGGGDuDkARRRKXI4laCHxHhY8dJy4wE/edit?usp=sharing) at the next office hours.
+
+## Helios Feedback
```
Comment (didoo) [inline website/docs/about/support.md:33]:
```suggestion
## Share your feedback
```

or we could consider even `Share your feedback anonymously` which it's even more explicit.

**Review by didoo — APPROVED** [review]

**Review by jorytindall — APPROVED** [review]
Agree with @didoo 's points, otherwise :shipit:

File: website/app/components/doc/page/footer.hbs [hunk 3]
```diff
@@ -21,6 +21,9 @@
       <li class="doc-page-footer__nav-item">
         <a href="https://www.hashicorp.com/privacy" target="_blank" rel="noopener noreferrer">Privacy</a>
       </li>
+      <li class="doc-page-footer__nav-item">
+        <a href="https://go.hashi.co/hds-feedback" target="_blank" rel="noopener noreferrer">Share feedback</a>
```
Comment (Dhaulagiri) [inline website/app/components/doc/page/footer.hbs:25]:
```suggestion
        <a href="https://go.hashi.co/hds-feedback" target="_blank" rel="noopener noreferrer">Share Feedback</a>
```

File: website/docs/about/support.md (hunk 2 above)
Comment (Dhaulagiri) [inline website/docs/about/support.md:33]:
```suggestion
## Share Feedback
```

Comment (Dhaulagiri) [reply website/docs/about/support.md]: the anonymous aspect is referenced in the copy below but I did update this to `Share Feedback`

**Pull request merged**